Graphics/Implicit/Export/SymbolicFormats.hs

```diff
@@ -100,24 +100,29 @@ buildShared (Scale v obj) = call "scale" (bf <$> elements v) [build obj]
 
 buildShared (Mirror v obj) = callNaked "mirror" [ "v=" <> bvect v ] [build obj]
 
+-- TODO(sandy): 'r' is not a rounding parameter
```

Comment:
your todo here is a bit unclear?

Reply:
This function is removed in #364 so I wasn't too worried about it.